src/pretalx/common/templates/common/base.html

```diff
@@ -102,6 +103,11 @@ <h1>
             </div>
         </section>
         <footer>
+            {% if is_html_export %}
+            <div id="exporttimestamp" class="text-muted">
+                this is a static export generated at {% now "Y-m-d H:i T" %}
```

Comment:
I'd prefer starting with a capitalized letter, and an internationalized version.